docs/self-managed/zeebe-gateway-deployment/the-zeebe-gateway.md

```diff
@@ -0,0 +1,169 @@
+
+:::note Be aware
+If the gateway is running in the embedded mode, it will consume resources from the broker, which might impact the performance of the system.
+:::
+
+### Standalone
+
+Running the gateway in standalone mode means the gateway will be executed as its own application. This is the recommended way for production use cases, and it is the default (and only option) in the Helm charts. As mentioned, this allows separation of concerns, especially as the gateway can be scaled independently of the broker based on the current workload.
+
+## Configuration
+
+The Zeebe Gateway can be configured similarly to the broker via the `application.yaml` file or environment variables. Find a detailed `application.yaml` with comments [here](https://github.com/camunda/zeebe/blob/main/dist/src/main/config/gateway.yaml.template).
+
+In the following sections, we provide tables with environment variables, application properties, a description, and their corresponding default values. We also describe a few use cases for each type of configuration.
+
+The configuration properties follow some conventions, especially the types for byte sizes and time units, check out the [application.yaml](https://github.com/camunda/zeebe/blob/main/dist/src/main/config/gateway.yaml.template), which describes this in detail.
+
+For deploying purposes, it is easier to use environment variables. The following sections outline usage of these variables. As Helm is the recommended way to deploy Camunda Platform 8, we will explain some configuration options here as well.
+
+### Network configuration
+
+The network configuration allows configuration of the host and port details for the gateway.
+
+| Environment variable         | Application.yaml property     | Description                         | Default value |
+|------------------------------|-------------------------------|-------------------------------------|---------------|
+| `ZEEBE_GATEWAY_NETWORK_HOST` | `zeebe.gateway.network.host`  | Sets the host the gateway binds to. | `0.0.0.0`     |
+| `ZEEBE_GATEWAY_NETWORK_PORT` | `zeebe.gateway.network.port`  | Sets the port the gateway binds to. | `26500`       |
+| `ZEEBE_GATEWAY_NETWORK_MINKEEPALIVEINTERVAL` | `zeebe.gateway.network.minKeepAliveInterval` | This setting specifies the minimum accepted interval between keep alive requests. If clients send keep-alive requests at a smaller rate, they are forcefully closed by the gateway. | `30s` |
+
+### Cluster configuration
+
+As mentioned, the gateway needs to connect to the Zeebe brokers.
+
+It is important to configure the cluster contact point to one of the Zeebe brokers. Be aware that the gateway currently doesn’t support a list of contact points. The contact point needs to be available to the gateway. With the help of the SWIM protocol, the gateway will find the other broker nodes (if there are any). The corresponding environment variable is called `ZEEBE_GATEWAY_CLUSTER_CONTACTPOINT`.
```

Comment:
What is SWIM protocol?

Reply:
Sorry, I refering here to https://en.wikipedia.org/wiki/SWIM_Protocol is a derived form of the Gossip protocol.

Reply:
No need to apologize, I'm just curious. Do you feel most engineers/users reading this would be familiar with that term? If so, I won't add any details.